docker/ubuntu_20_04_lofar

```diff
@@ -1,30 +1,8 @@
-FROM ubuntu:20.04
-
-RUN export DEBIAN_FRONTEND=noninteractive && apt-get update && \
-    apt-get install -y \
-    libboost-python-dev libcfitsio-dev python3-dev python3-numpy cmake \
-    build-essential libhdf5-serial-dev libarmadillo-dev \
-    libboost-filesystem-dev libboost-system-dev libboost-date-time-dev \
-    libboost-program-options-dev libboost-test-dev \
-    libxml2-dev libpng-dev pkg-config \
-    libgtkmm-3.0-dev git libfftw3-dev \
-    gfortran flex bison wcslib-dev \
-    libboost-numpy-dev liblua5.3-dev \
-    casacore-dev aoflagger-dev
-
-# Build latest IDG master from source
-RUN mkdir /idg && cd /idg && git clone https://gitlab.com/astron-idg/idg.git src
-RUN mkdir /idg/build && cd /idg/build && cmake -DCMAKE_INSTALL_PREFIX=/usr ../src && make install -j2
-
-# Build LOFARBeam
-RUN mkdir /lofarbeam && cd /lofarbeam && git clone https://github.com/lofar-astron/LOFARBeam.git src
-RUN mkdir /lofarbeam/build && cd /lofarbeam/build && cmake -DCMAKE_INSTALL_PREFIX=/usr ../src -DPYTHON_EXECUTABLE=/usr/bin/python3 && make install -j2
-
-ADD . /src
-WORKDIR /src
-
-RUN mkdir /build && cd /build && cmake ../src
-RUN cd /build && make -j2
-RUN cd /build && make install -j2
-RUN DPPP
-RUN cd /build && CTEST_OUTPUT_ON_FAILURE=1 make test -j2
+FROM dppp_base:20.04.0
+
```

Comment:
In view of previous comment: here you want to `ADD` the `src/` directory (and set the `WORKDIR`)